scripts/release/rpm/azure-cli.spec

```diff
@@ -2,8 +2,14 @@
 # Definition of macros used - https://fedoraproject.org/wiki/Packaging:RPMMacros?rd=Packaging/RPMMacros
 
 # .el7.centos -> .el7
-%if 0%{?rhel} == 7
-  %define dist .el7
+%if 0%{?rhel}
+  %define dist .el%{?rhel}
+%endif
+
+%if 0%{?rhel} < 8
+    %define python_cmd python2
```

Comment:
Just a question, guess older RHEL doesn't have Python3 pre-installed?

Reply:
Correct, and worse yet it's not even available in the standard repositories. You can test this out by running the following command:
``` PowerShell
docker run --rm -it centos:7 yum install -y python3
```

There are ways to get at it, but you need to use an SCL appropriate for the EL7 distro your using as seen here: https://linuxize.com/post/how-to-install-python-3-on-centos-7/

Reply:
Thinking about it more, I do think I'd be able to syntactically require python3 and get the installation right. However, I just don't know enough about running an EL7 instance to know if people are open to using the SCLs or not.

Reply:
I think this PR is good enough to get started. Still new to EL world, but I guess we can drop support for EL7 around the time that Python 2 gets deprecated?

Reply:
IMHO, we can safely take the position that no new versions of the CLI will be supported on EL7 starting in January. However, we may need to publish security patches from time to time.